src/StrawberryShake/Client/src/Transport.WebSockets/WebSocket/MessageHandler.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using StrawberryShake.Transport;
+using StrawberryShake.Transport.WebSockets.Messages;
+
+namespace StrawberryShake.Http.Subscriptions
+{
+    public abstract class MessageHandler<T>
```

Comment:
Didnt change much in the pipeline, something i will look at when the stuff with the socketclient, operationmanager and protocol is resolved